src/Raven.Server/Documents/Handlers/Admin/RachisAdminHandler.cs

```diff
@@ -203,7 +203,7 @@ public Task AllowPreflightReuqest()
 
                         var nodeTag = nodeInfo.NodeTag == "?" 
                             ? null : nodeInfo.NodeTag;
-                        await ServerStore.AddNodeToClusterAsync(serverUrl, Convert.FromBase64String(nodeInfo.PublicKey), nodeTag, validateNotInTopology:false);
+                        await ServerStore.AddNodeToClusterAsync(serverUrl, Convert.FromBase64String(nodeInfo.PublicKey ?? string.Empty), nodeTag, validateNotInTopology:false);
```

Comment:
Please look into exactly what is going on here. This is not a good idea for us to do.